src/main/java/org/folio/rs/controller/ConfigurationsController.java

```diff
@@ -2,37 +2,37 @@
 
 import static java.util.Objects.isNull;
 
+import java.util.List;
+import javax.persistence.EntityNotFoundException;
+import javax.validation.Valid;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.log4j.Log4j2;
+import org.folio.rs.domain.dto.Provider;
 import org.folio.rs.domain.dto.StorageConfiguration;
 import org.folio.rs.domain.dto.StorageConfigurations;
 import org.folio.rs.rest.resource.ConfigurationsApi;
 import org.folio.rs.service.ConfigurationsService;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.folio.rs.service.ProviderService;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.persistence.EntityNotFoundException;
-import javax.validation.Valid;
-import javax.validation.constraints.Max;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.Pattern;
-
 @Log4j2
 @RestController
+@RequiredArgsConstructor
 @RequestMapping(value = "/remote-storage/")
 public class ConfigurationsController implements ConfigurationsApi {
   private static final String CONFIGURATION_NOT_FOUND = "Configuration not found";
 
   private final ConfigurationsService configurationsService;
-
-  @Autowired
-  public ConfigurationsController(ConfigurationsService configurationsService) {
-    this.configurationsService = configurationsService;
-  }
+  private final ProviderService providerService;
```

Comment:
Please, rename ProvidersService just to be consistent.

Reply:
Renamed